lib/main.dart

```diff
@@ -16,16 +17,12 @@ var logger;
 
 void main() {
   runApp(const MyApp());
-  requestPermissions();
-  SessionManager().loadSessionsFromJson();
   // If in Debug Mode this Code will be executed
   // Else this code will be removed automatically
   if (kDebugMode) {
```

Comment:
muss eh weg mach ich noch in einem finalen refactoring und geht den Code durch mit den Coding Convetions